docs/compass/03-automatic-scenario-assignment.md

```diff
@@ -0,0 +1,213 @@
+# Automatic Scenario Assignment
+
+To connect and group your Applications and Runtimes, assign them to the same scenario.
+Automatic Scenario Assignment feature allows defining condition when a Scenario is automatically assigned to the Runtime.
```

Comment:
We can specify what the condition means - a label selector). Then we could have here 2 sentences, which is good enough for an introduction.

Reply:
I would not limit the feature to runtimes. We could rephrase it and write in another sentence that only Runtimes are supported for now.